src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -462,16 +460,8 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
                     setSweepOpen(!sweepIsOpen)
                   }}
                 >
-                  <SweepIcon width="24px" height="24px" />
-                  <ThemedText.BodyPrimary
-                    fontWeight={600}
-                    color={sweepIsOpen && buyNow ? theme.white : theme.textPrimary}
-                    lineHeight="20px"
-                    marginTop="2px"
-                    marginBottom="2px"
-                  >
-                    Sweep
-                  </ThemedText.BodyPrimary>
+                  <SweepIcon viewBox="0 0 24 24" width="20px" height="20px" />
```

Comment:
i think the viewbox could be set based on width/height

Reply:
nah they kinda vary across SVGs...there is not a uniform appropriate viewBox across icons. kind of annoying but we really have to tweak them individually to get it to look right.